src/main/java/animals/Animals.java

```diff
@@ -1,15 +1,15 @@
 package animals;
 
-public class Animals {
+public abstract class Animals {
```

Comment:
Here and in all other classes better name the class in singular form (eg. Animal, Bird etc.) as the object made on the base of those classes is a single animal/bird/whatever, not a list of them.